pkgs/applications/science/logic/isabelle/default.nix

```
@@ -1,5 +1,24 @@
-{ lib, stdenv, fetchurl, coreutils, nettools, java, scala, polyml, z3, veriT, vampire, eprover-ho, naproche, rlwrap, perl, makeDesktopItem, isabelle-components, isabelle, symlinkJoin, fetchhg }:
-# nettools needed for hostname
+{ lib
+, stdenv
+, fetchurl
+, coreutils
+, nettools # nettools needed for hostname
```

```suggestion
, nettools
```
Please move the comment to where hostname is used.